gerberimporter.hpp

```diff
@@ -29,6 +30,12 @@
 
 extern "C" {
 #include <gerbv.h>
+
+// This depends on GERBV_INTERPOLATION_LINEARx10 not being the first item
+// in the enum.
+#if COMPILE_VALUE_NEW_LINEARx10==0
```

Comment:
The `x` needs to be a capital `X` due to some transformation that is done in the m4 file.